### PR TITLE
Add week1 content for portfolio page

### PR DESCRIPTION
**Pull request by lolfuljames**
This commit adds content and style to the page, including images and icon needed.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,18 +1,291 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8" />
+    <title>Home - James Tan</title>
+    <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
```
Comment (henryken) [inline portfolio/src/main/webapp/index.html]: To improve the website performance better, you can read an article below
https://flaviocopes.com/javascript-async-defer/

**Review by henryken — COMMENTED** [review]
@lolfuljames, I have informational comment to improve performance better. See if you can read and understand the article below.
https://flaviocopes.com/javascript-async-defer/

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (lolfuljames) [reply portfolio/src/main/webapp/index.html]: > To improve the website performance better, you can read an article below
> https://flaviocopes.com/javascript-async-defer/

Noted, I'll be adding defer along with some of the minor tweaks to improve the ergonomics of the website..

**Review by henryken — APPROVED** [review]
Great work!

**Pull request merged**